app/src/lib/oauth.ts

```diff
@@ -40,11 +40,12 @@ export function askUserToOAuth(endpoint: string) {
  */
 export async function requestAuthenticatedUser(
   code: string
-): Promise<Account | null> {
+): Promise<Account | null | undefined> {
```

Comment:
This was necessary to prevent the original code path from considering "this callback is useless" the same as "i have a token but it doesn't work"  and displaying this error in the dev tools:

<img width="686" src="https://user-images.githubusercontent.com/359239/43793352-a6f7325e-9a51-11e8-883a-06f9e40ad1ea.png">

Reply:
Would you mind updating the doc above with what the difference between `null` and `undefined` is?